application/helpers/update/updates/Update_629.php

```diff
@@ -21,7 +21,7 @@ public function up(): void
         $this->db->createCommand()->update(
             '{{user_mfa_settings}}',
             ['authType' => 'totp'],
```

Comment:
You do not need it here ?

Reply:
Nope, not needed there. That is solved by the builder.

Reply:
Below is needed as the condition is passed as plain text